worker/src/RTC/RtpStreamSend.cpp

```diff
@@ -10,34 +10,146 @@ namespace RTC
 {
 	/* Static. */
 
+	thread_local static Utils::ObjectPool<RtpStreamSend::StorageItem> StorageItemPool;
+
 	// 17: 16 bit mask + the initial sequence number.
 	static constexpr size_t MaxRequestedPackets{ 17 };
 	thread_local static std::vector<RTC::RtpStreamSend::StorageItem*> RetransmissionContainer(
 	  MaxRequestedPackets + 1);
+	static constexpr uint32_t MinRetransmissionDelay{ 200 };
 	// Don't retransmit packets older than this (ms).
 	static constexpr uint32_t MaxRetransmissionDelay{ 2000 };
 	static constexpr uint32_t DefaultRtt{ 100 };
+	static constexpr uint16_t MaxSeq = std::numeric_limits<uint16_t>::max();
+	static constexpr uint32_t MaxTs  = std::numeric_limits<uint32_t>::max();
 
 	static void resetStorageItem(RTC::RtpStreamSend::StorageItem* storageItem)
 	{
 		MS_TRACE();
 
 		MS_ASSERT(storageItem, "storageItem cannot be nullptr");
 
-		delete storageItem->packet;
-
-		storageItem->packet     = nullptr;
+		storageItem->clonedPacket.Reset();
```

Comment:
I know this has not been changed in this PR, but this method `resetStorageItem` feels like it should be a member method `StorageItem::Reset()` rather than a static method.